dataclients/kubernetes/definitions/testdata/validation/duplicate-default-backend.log

```diff
@@ -0,0 +1,2 @@
+test-route-group
+duplicate backend reference: app
```

Comment:
2 lines look a bit weird, do I miss something?

Reply:
The fixture test treats each line as a regexp and matches it against actual log lines.
I made it similar to other golden log files but now I removed the first line as we are only interested to check the second one.